project/src/const.js

```diff
@@ -0,0 +1,6 @@
+export const AppRoute = {
+  MAIN: '/',
+  SIGN_IN: '/login',
+  FAVORITES: '/favorites',
+  ROOM: '/offer/:id',
+};
```

Comment:
Экспорты должны быть внизу файла
```suggestion
const AppRoute = {
  MAIN: '/',
  SIGN_IN: '/login',
  FAVORITES: '/favorites',
  ROOM: '/offer/:id',
};

export { AppRoute };
```